gradlew.bat

```diff
@@ -0,0 +1,92 @@
+@rem

```

Comment:
I do not see us ever needing to run the gradle related operations on Windows, but who knows.

Reply:
The shell script works in recent Windows versions, but I'd keep the bat-file for convenience.

https://github.com/Aiven-Open/commons-for-apache-kafka-connect/blob/f579c4c9f2ef2927be04fe8f16592b4a17718986/.github/workflows/main_push_workflow.yml#L25-L27